src/components/events/commonEventFields/eventDetails.tsx

```diff
@@ -1,26 +1,26 @@
 import { useState } from 'react';
 import { IEvent } from '../../../types/interfaces';
 import {
-  additionalEventDetails,
-  eventFlightDetails,
+  ADDITIONAL_EVENT_DETAILS,
+  EVENT_FLIGHT_DETAILS,
 } from '../../../types/configurations';
 import AdditionalDetails from './additionalDetails';
 
-interface EventProps {
+interface Props {
     disabled: boolean;
     isValue: boolean;
     setAttr: (attr: keyof IEvent, value: any) => void;
     event: IEvent;
 }
 
-const EventDetails = ({ disabled, isValue, event, setAttr }: EventProps) => {
+const EventDetails = ({ disabled, isValue, event, setAttr }: Props) => {
   const [displayFlight, setDisplayFlight] = useState<boolean>(false);
```

Comment:
define the state inside the inner component?

Reply:
It is a general component and the states should be separate.
If I would define it inside both components would always open and close together.